spec/requests/authentication_spec.rb

```diff
@@ -202,6 +202,48 @@
     end
   end
 
+  context "UI authentication against the rails session" do
```

Comment:
Instead of allows here, can we change them to expect to ensure they are called, in case code changes in the future?

Reply:
:+1:

Reply:
Except of one, this is doable, the `:[]` mock has to stay as it is because it can be called multiple times and anywait, having an expect would not add any value into this.